torch/backends/cuda/__init__.py

```diff
@@ -83,16 +83,15 @@ def __setattr__(self, name, value):
             return super(cuFFTPlanCacheManager, self).__setattr__(name, value)
 
 
-class CUDAModule(object):
-    def __init__(self, m):
-        self.__dict__ = m.__dict__
-        # You have to retain the old module, otherwise it will
-        # get GC'ed and a lot of things will break.  See:
-        # https://stackoverflow.com/questions/47540722/how-do-i-use-the-sys-modules-replacement-trick-in-init-py-on-python-2
-        self.__old_mod = m
-
-    cufft_plan_cache = cuFFTPlanCacheManager()
-
-# This is the sys.modules replacement trick, see
-# https://stackoverflow.com/questions/2447353/getattr-on-a-module/7668273#7668273
-sys.modules[__name__] = CUDAModule(sys.modules[__name__])
+class cuBLASModule:
```

Comment:
what happened to CUDAModule?

Reply:
This line below `sys.modules[__name__] = CUDAModule(sys.modules[__name__])` replaces the module with an object of `cuBLASModule`, I don't see any need for doing so, and mypy can not correctly find the `matmul` if doing so.